cache-config/t3c-apply/t3c-apply.go

```diff
@@ -61,7 +60,7 @@ func main() {
 	var lock util.FileLock
 	cfg, err := config.GetCfg()
 	if err != nil {
-		fmt.Println(err)
+		log.Errorln(err)
```

Comment:
This error happened because the config was malformed, so the loggers won't exist.
I think it needs to be a `fmt.Println`, there's just no way around it here. 

And we need the error message, otherwise it's going to be impossible to figure out why it's failing.

Reply:
Put back to fmt.Println